src/main/java/com/cinema/model/dto/MovieDto.java

```diff
@@ -0,0 +1,9 @@
+package com.cinema.model.dto;
+
+import lombok.Data;
+
+@Data
+public class MovieDto {
```

Comment:
let's use two dtos: request (without id) and response(with movie id)

Reply:
done